nss-cohort-36/bangazon-client-virtual-magicians-1
# Jl order react

# Description
[Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. List any dependencies that are required for this change.]
Closes #[Enter issue #]
## Type of change

Please delete options that are not relevant.

# Testing Instructions
Make sure to commit local changes in your branch before switching to master.
1. `git checkout master`
2. `git fetch --all`
3. `git checkout jl-order-react`
4. after running migrations on api side:
5. click buy in navbar and you will see 22 products
6. products are hyperlinked to details view 
7. details view allows buy button to work - click buy and you will be routed to shopping cart
8. you can log out and register a new user (will need to manually go to /register)
9. follow above instructions and you will be able to create a new order and see shopping cart

# Checklist:
